python/social_bot/envs/grocery_ground.py

```diff
@@ -81,12 +108,13 @@ def run(self, agent, world):
                 steps_since_last_reward = 0
             else:
                 if self._reward_shaping:
-                    agent_sentence = yield TeacherAction(
-                        reward=-dist / self._random_range,
-                        sentence="Failed",
-                        done=False)
+                    reward = -dist / self._random_range
                 else:
-                    agent_sentence = yield TeacherAction()
+                    reward=0.0
+                agent_sentence = yield TeacherAction(
+                    reward=reward,
+                    sentence="Please go to "+self._goal_name,
```

Comment:
Right now it's ok to only use the goal name in the sentence (i.e. without "Please go to"). We'll do full sentences later.

Reply:
OK